README.md

```diff
@@ -2,17 +2,17 @@ Xenia - Xbox 360 Emulator Research Project
 ==========================================
 
 Xenia is an experimental emulator for the Xbox 360. For more information see the
-[main xenia website](https://xenia.jp/).
+[main Xenia website](https://xenia.jp/).
 
 **Interested in supporting the core contributors?
 [Xenia Project on Patreon](https://www.patreon.com/xenia_project).**
 
-Come chat with us about **emulator-related topics** on [Discord](https://discord.gg/Q9mxZf9).
+Come chat with us about **emulator-related topics** on [Discord](https://discord.me/xenia-emulator).
```

Comment:
Please undo this change. I didn't notice the `.me` TLD change and this page has ads.